Dockerfile

```
@@ -15,3 +15,5 @@ COPY package.json ./
 ENV PG_META_PORT=8080
 CMD ["npm", "run", "start"]
 EXPOSE 8080
+# --start-period defaults to 0s, but can't be set to 0s (to be explicit) by now
+HEALTHCHECK --interval=5s --timeout=5s --retries=3 CMD curl -f http://localhost:8080/health || exit 1
```

Are you sure curl exists in this docker image?

```bash
root@c4341b754f88:/usr/src/app# curl --version
bash: curl: command not found
```

---

Good catch - let's use this command instead: https://github.com/supabase/cli/blob/ab821f164ea0bbe88d27c4e1e39f024467c2cbd7/internal/start/start.go#L403

---

If you don't want to install curl as a runtime dependency, an alternative is to use [node](https://github.com/supabase/cli/blob/main/internal/start/start.go#L403) for health check.

```suggestion
HEALTHCHECK --interval=5s --timeout=5s --retries=3 CMD node -e "require('http').get('http://localhost:8080/health', (r) => {if (r.statusCode !== 200) throw new Error(r.statusCode)})"
```

---

Good catch, didn't notice it was slim.
Agree with node version of the healthcheck cmd